pkg/tsdb/prometheus/time_series_query.go

```diff
@@ -262,6 +254,20 @@ func calculateRateInterval(interval time.Duration, scrapeInterval string, interv
 	return rateInterval
 }
 
+func interpolateVariables(expr string, interval time.Duration, timeRange time.Duration, intervalCalculator intervalv2.Calculator, timeInterval string) string {
+	rangeMs := timeRange.Milliseconds()
+	rangeSRounded := int64(math.Round(float64(rangeMs) / 1000.0))
+	fmt.Println(rangeMs, rangeSRounded)
```

Comment:
you probably want to remove this `fmt.Println`...

Reply:
```suggestion
```

Reply:
Yes!  Good catch. Removed